explicitly checkAndGetColumn for remaining H3 funcs

bharatnc opened the pull request:
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Explicitly checkAndGetColumn for the remaining H3 funcs.

cc: @kitaisreal

bharatnc committed:
h3 explicitly checkAndGetColumn

bharatnc committed:
h3 remaining funcs checkAndGetColumn

bharatnc committed:
fixes to exception message

bharatnc committed:
fix build

bharatnc committed:
better arg check for h3kRing func

kitaisreal commented:
01442_h3kring_range_check need to fix, probably for second argument we need to specify exact type, not just any integer.

bharatnc committed:
fix tests

bharatnc commented:
> 01442_h3kring_range_check need to fix, probably for second argument we need to specify exact type, not just any integer.

Yes, fixed. Also changed to check explicitly for UInt16 for second arg k since it always should be > 0 < 10000.
